src/main/java/com/google/cloud/tools/appengine/v2/internal/cloudsdk/deploy/CloudSdkDeployTranslator.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.appengine.v2.internal.cloudsdk.deploy;
+
+import com.google.cloud.tools.appengine.api.deploy.DeployConfiguration;
+import com.google.cloud.tools.appengine.cloudsdk.internal.args.GcloudArgs;
+import com.google.cloud.tools.appengine.v2.internal.gcloud.ConfigurationTranslator;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Lists;
+
+import java.io.File;
+import java.util.List;
+
+public class CloudSdkDeployTranslator implements ConfigurationTranslator<DeployConfiguration> {
```

Comment:
Do we really need this extra complexity? What are the use cases for using other translators that return string lists?